README.md

```diff
@@ -47,26 +52,35 @@ SSB-JS organization require discussion and consent.
 - Projects SHOULD have automated tests for all target platforms.
 - Projects SHOULD have internally consistent contribution guidelines.
 
-### Contributing
+### Contributors
+
+- 'Contributors' are anyone who participates in a project.
+- Contributors MUST aim to foster an open, welcoming, and harassment-free space.
+- Contributors SHOULD recommend changes in the form of patches.
+- Contributors who want to be maintainers SHOULD participate consistently and often.
+
+### Maintainers
 
-- Anyone MUST be able to propose changes to repositories via pull request.
-- Anyone who is not a maintainer MUST NOT have the ability to push commits to repositories.
-- Maintainers MUST have the ability to push commits to every repository.
+- 'Maintainers' are contributors trusted to steward a project and vote.
```

Comment:
what is "vote" - do you mean signal approval for changes to governance?

Reply:
Oh, nice catch. That's exactly what I meant. Could certainly use a sentence refactor.